File: powerbi_python_r_scripts/python/scatter_plot.py
import matplotlib.pyplot as plt
df = dataset
plt.figure(figsize=(8, 5))
plt.scatter(df['Idade'], df['Salário'], alpha=0.7)
plt.title('Idade vs Salário')
plt.xlabel('Idade')
plt.ylabel('Salário')
plt.grid(True)
plt.tight_layout()
plt.show()

File: powerbi_python_r_scripts/r/scatter_plot.R
library(ggplot2)
df <- dataset
ggplot(df, aes(x = Idade, y = Salario)) +
  geom_point(alpha = 0.7, color = "blue") +
  theme_minimal() +
  labs(title = "Idade vs Salário", x = "Idade", y = "Salário")